xayn-ai/src/ranker/config.rs

```diff
@@ -15,6 +15,8 @@ pub(crate) struct Configuration {
     max_key_phrases: usize,
     gamma: f32,
     penalty: Vec<f32>,
+    min_positive_cois: u32,
+    min_negative_cois: u32,
```

Comment:
is there a specific reason that those two are `u32`? otherwise we could make them `usize` as usual.

Reply:
No specific reason. I will replace it with `usize`

Reply:
done